apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet.Tests/ErrorReporting/ErrorReportingUtils.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+
+using System;
+
+namespace Google.Cloud.Diagnostics.AspNet.Tests
+{
+    internal static class ErrorReportingUtils
+    {
+        /// <summary>Returns true if the current OS is Windows.</summary>
+        public static bool IsWindows() => Environment.OSVersion.ToString().Contains("Windows");
```

Comment:
Not that important since its internal: I would probably make this a read-only property instead of a method.